experimental/addons/crossplane-aws/metadata.yaml

```diff
@@ -0,0 +1,15 @@
+name: crossplane-aws
+version: 0.0.1
+description: AWS Provider for Crossplane
+icon: https://static.kubevela.net/images/logos/aws.png
+url: https://crossplane.io/
+
+tags:
+  - Crossplane provider
```

Comment:
“- cloud resource”？align with terraform.

Reply:
Yes, I did in addon `crossplane` https://github.com/kubevela/catalog/pull/376/files#diff-1f030f14d0e749a3b151ca6f5240a231928dd54545f64b5230f01ed4c5dcc387R8.